src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -728,7 +728,7 @@ oop InstanceKlass::protection_domain() const {
   return java_lang_Class::protection_domain(java_mirror());
 }
 
-// To remove these from requires an incompatible change and CCC request.
+// To remove these from requires an incompatible change and CSR review.
```

Comment:
I don't know what this comment is trying to say; I think there might be missing words or something.  But the change for CCC -> CSR is fine.

Reply:
Given the 'R' in CSR already stands for Review this should have said "CSR request".

But I also have no idea what the comment is actually trying to say - what is "these" referring to???

Reply:
I don't know why that comment is there. The API is Class::getSigners and any changes to its behavior would require a CSR, but we are free to change the implementation. So maybe the comment should be removed.

Reply:
Filed JDK-8276889 in case further cleanup of the wording in instanceKlass.cpp is desired.

Reply:
Oh at one point we were trying to figure out a different way of implementing signers so that it didn't have to store a field per InstanceKlass, when we were working on density.